app/src/main/java/com/jienihou/outfitxchange/HomeScreen.kt

```diff
@@ -15,11 +23,52 @@ import androidx.fragment.app.Fragment
  * create an instance of this fragment.
  */
 class HomeScreen : Fragment() {
+    private lateinit var bottomNavigationView: BottomNavigationViewEx
+    private lateinit var categoryAdapter: CategoryRVAdapter
+
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_home_screen, container, false)
+        val view = inflater.inflate(R.layout.fragment_home_screen, container, false)
+
+        // Logo bar
+         val toolbar = view?.findViewById(R.id.toolbar) as Toolbar
+        (activity as AppCompatActivity).setSupportActionBar(toolbar)
+
+        // Categories Grid Recycler View
+        val categoryRV = view.findViewById(R.id.rv_categories) as RecyclerView
+        val columns = 2
+        categoryRV.layoutManager = GridLayoutManager(this.context, columns)
```

Comment:
I think a good pattern to use when setting multiple things on an object is kotlin's [apply](https://kotlinlang.org/docs/reference/scope-functions.html#apply) scope function.

e.g

```
categoryRV.apply {
  adapter = categoryAdapter,
  layoutManager = GridLayoutManager(this.context, columns)
}